pages/api/recipes.tsx

```diff
@@ -0,0 +1,14 @@
+import { connectToDatabase } from "../../lib/mongodb";
+
+export default async (req, res) => {
+  const { db } = await connectToDatabase();
+
+  const recipes = await db
+    .collection("cocktaile_recipes")
```

Comment:
Typo

Reply:
Σωστό είναι αυτό, αφού τα βλέπουμε τα αποτελέσματα. To typo έγινε στην database :P